### PR TITLE
Live Code Example

### PR DESCRIPTION
**Pull request by SilentCicero**
Code example and welcome cleanup.

### CODE REVIEW

File: src/v1.0.0/1. Introduction/1. Welcome.md [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 ===
 
-[![Community](https://img.shields.io/badge/chat%20on-discord-orange?&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/xfpK4Pe)
+Before we begin:
 
-🔍 Rinkeby testnet block explorer: [https://rinkeby.fuel.sh/network](https://rinkeby.fuel.sh/network).
+Try out our live interactive [🕹️ Fuel Plays Pokémon demo!](https://fuelplayspokemon.com)
 
-📺 [Try out our live interactive Fuel Plays Pokémon demo!](https://fuelplayspokemon.com)
+Checkout the [⚡ Block Explorer](https://rinkeby.fuel.sh/network)
```
Comment (adlerjohn) [inline src/v1.0.0/1. Introduction/1. Welcome.md:8]:
```suggestion
Check out the [⚡ Block Explorer](https://rinkeby.fuel.sh/network).
```

File: src/v1.0.0/1. Introduction/1. Welcome.md [hunk 2]
```diff
@@ -1,13 +1,17 @@
 Welcome
 ===
 
-[![Community](https://img.shields.io/badge/chat%20on-discord-orange?&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/xfpK4Pe)
+Before we begin:
 
-🔍 Rinkeby testnet block explorer: [https://rinkeby.fuel.sh/network](https://rinkeby.fuel.sh/network).
+Try out our live interactive [🕹️ Fuel Plays Pokémon demo!](https://fuelplayspokemon.com)
```
Comment (adlerjohn) [inline src/v1.0.0/1. Introduction/1. Welcome.md:6]:
```suggestion
Try out our live interactive [🕹️ Fuel Plays Pokémon demo!](https://fuelplayspokemon.com).
```

File: src/v1.0.0/1. Introduction/1. Welcome.md [hunk 3]
```diff
@@ -1,13 +1,17 @@
...
 
-📺 [Try out our live interactive Fuel Plays Pokémon demo!](https://fuelplayspokemon.com)
+Checkout the [⚡ Block Explorer](https://rinkeby.fuel.sh/network)
 
-💸 [Demo wallet with WalletConnect and social media integration!](https://redditcash.fuel.sh)
+Use the one-click [💻 Wallet SDK Example](/v1.0.0/Getting%20Started/One-Click%20Start.html)
+
+We are always available to...
+
+[![Community](/assets/images/discord.svg)](https://discord.gg/xfpK4Pe)
```
Comment (adlerjohn) [inline src/v1.0.0/1. Introduction/1. Welcome.md:14]:
```suggestion
We are always available to... [![Community](/assets/images/discord.svg)](https://discord.gg/xfpK4Pe)
```

File: src/v1.0.0/2. Getting Started/0. One-Click Start.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+const provider = null; // window.web3.currentProvider
+
+const wallet = new fuel.Wallet(provider, {
+  key: 'new',
+  network: 'rinkeby',
+});
+
+await wallet.faucet();
+</div>
+-->
```
Comment (adlerjohn) [inline src/v1.0.0/2. Getting Started/0. One-Click Start.md:157]:
```suggestion
```

File: src/v1.0.0/2. Getting Started/0. One-Click Start.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
+One-Click Start
+===
+
+Below is our one-click start example using the Fuel wallet and the fuel test faucet on Rinkeby.
```
Comment (adlerjohn) [inline src/v1.0.0/2. Getting Started/0. One-Click Start.md:4]:
```suggestion
Below is our one-click start example of using the Fuel wallet and the Fuel test faucet on Rinkeby.
```

**Comment by SilentCicero** [comment]
Fixed links, new code highlighting, available for viewing here:

https://docsprivate.fuel.sh/v1.0.0/Introduction/Welcome.html

File: public/index.css [hunk 6]
```diff
@@ -46,13 +58,64 @@ summary {
...
+
+pre .line-number span {
+  display:block;
+  padding:0 .5em 0 1em;
+}
+
+pre .cl {
+  display:block;
+  clear:both;
+} */
```
Comment (adlerjohn) [inline public/index.css:103]:
```suggestion
```

File: src/v1.0.0/1. Introduction/1. Welcome.md [hunk 7]
```diff
@@ -1,13 +1,15 @@
 Welcome
 ===
 
-[![Community](https://img.shields.io/badge/chat%20on-discord-orange?&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/xfpK4Pe)
+**Before we begin:**
 
-🔍 Rinkeby testnet block explorer: [https://rinkeby.fuel.sh/network](https://rinkeby.fuel.sh/network).
+Try out our live interactive [🕹️ Fuel Plays Pokémon demo!](https://fuelplayspokemon.com).
```
Comment (adlerjohn) [inline src/v1.0.0/1. Introduction/1. Welcome.md:6]:
```suggestion
Try out our live interactive [🕹️ Fuel Plays Pokémon demo](https://fuelplayspokemon.com)!
```

File: src/v1.0.0/1. Introduction/1. Welcome.md [hunk 8]
```diff
@@ -1,13 +1,15 @@
...
 
-📺 [Try out our live interactive Fuel Plays Pokémon demo!](https://fuelplayspokemon.com)
+Check out the [⚡ Block Explorer](https://rinkeby.fuel.sh/network).
 
-💸 [Demo wallet with WalletConnect and social media integration!](https://redditcash.fuel.sh)
+Use the one-click [💻 Wallet SDK Example](../2.%20Getting%20Started/1.%20One-Click%20Example.md)
```
Comment (adlerjohn) [inline src/v1.0.0/1. Introduction/1. Welcome.md:10]:
```suggestion
Use the one-click [💻 Wallet SDK Example](../2.%20Getting%20Started/1.%20One-Click%20Example.md).
```

File: src/v1.0.0/1. Introduction/3. Use Cases.md [hunk 9]
```diff
@@ -0,0 +1,62 @@
+Use Cases
+===
+
+In this section we explore various use-cases for the Fuel Version 1.0 ORU. All the use-cases covered below are examined in more detail in the [Tools and Applications](../4.%20Tools%20and%20Applications/0.%20Fast%20Withdrawals.md) section.
```
Comment (adlerjohn) [inline src/v1.0.0/1. Introduction/3. Use Cases.md:4]:
```suggestion
In this section we explore various use cases for the Fuel [v1](../Future%20Roadmap/Planned%20Features.md#fuelv1) optimistic rollup. All the use cases covered below are examined in more detail in the [Tools and Applications](../4.%20Tools%20and%20Applications/0.%20Fast%20Withdrawals.md) section.
```

File: src/v1.0.0/1. Introduction/3. Use Cases.md [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+
+Transfers
+---
+
+Fuel can be used for high-performance, low-latency global real-time transfers using ERC20 Ethereum compliant tokens.
```
Comment (adlerjohn) [inline src/v1.0.0/1. Introduction/3. Use Cases.md:9]:
```suggestion
Fuel can be used for high-performance, low-latency global transfers with any ERC-20 token or Ether.
```

File: src/v1.0.0/1. Introduction/3. Use Cases.md [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+
+**This is ideal for:**
+- Global payment settlement
+- Application specific tokens
```
Comment (adlerjohn) [inline src/v1.0.0/1. Introduction/3. Use Cases.md:13]:
```suggestion
- Application-specific tokens
```

File: src/v1.0.0/1. Introduction/3. Use Cases.md [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+- Virtual gaming tokens and rewards
+- Any situations where users are transferring ERC20's
```
Comment (adlerjohn) [inline src/v1.0.0/1. Introduction/3. Use Cases.md:15]:
```suggestion
- Any situations where users are transferring ERC-20 tokens
```

File: src/v1.0.0/1. Introduction/3. Use Cases.md [hunk 13]
```diff
@@ -0,0 +1,62 @@
...
+
+Please see our interactive [🕹️ Fuel Plays Pokémon demo](https://fuelplayspokemon.com) for a live example of transfers in action.
+
+Non-Custodial Exchange
+---
+
+Fuel's unique **multi-signature transactions and hash-time-lock contracts** are a perfect foundation for non-custodial exchanges.
```
Comment (adlerjohn) [inline src/v1.0.0/1. Introduction/3. Use Cases.md:22]:
```suggestion
Fuel's unique **multi-user transactions and HTLC support** are a perfect foundation for non-custodial exchanges.
```

File: src/v1.0.0/1. Introduction/3. Use Cases.md [hunk 14]
```diff
@@ -0,0 +1,62 @@
...
+
+**We support two kinds of exchange:**
+- Multi-signature Swaps
```
Comment (adlerjohn) [inline src/v1.0.0/1. Introduction/3. Use Cases.md:25]:
```suggestion
- Multi-user on-Fuel swaps
```

File: src/v1.0.0/1. Introduction/3. Use Cases.md [hunk 15]
```diff
@@ -0,0 +1,62 @@
...
+- HTLC Cross-Chain Atomic Swaps
```
Comment (adlerjohn) [inline src/v1.0.0/1. Introduction/3. Use Cases.md:26]:
```suggestion
- Cross-chain atomic swap with HTLCs
```

File: src/v1.0.0/1. Introduction/3. Use Cases.md [hunk 16]
```diff
@@ -0,0 +1,62 @@
...
+
+For further reading, please see [Non-Custodial Exchanges](../4.%20Tools%20and%20Applications/3.%20Non-Custodial%20Exchanges.md)
```
Comment (adlerjohn) [inline src/v1.0.0/1. Introduction/3. Use Cases.md:28]:
```suggestion
For further reading, please see [Non-Custodial Exchanges](../4.%20Tools%20and%20Applications/3.%20Non-Custodial%20Exchanges.md).
```

File: src/v1.0.0/1. Introduction/3. Use Cases.md [hunk 17]
```diff
@@ -0,0 +1,62 @@
...
+
+Mass-Token Dispersal
```
Comment (adlerjohn) [inline src/v1.0.0/1. Introduction/3. Use Cases.md:30]:
```suggestion
Mass Token Dispersal
```

File: src/v1.0.0/1. Introduction/3. Use Cases.md [hunk 18]
```diff
@@ -0,0 +1,62 @@
...
+---
+
+Fuel supports *multi-input, multi-output transactions*, making it perfect for pre-signed mass token dispersal scenarios where a *single account must disperse many-to-millions of tokens* to unique accounts cost-efficiently.
```
Comment (adlerjohn) [inline src/v1.0.0/1. Introduction/3. Use Cases.md:33]:
```suggestion
Fuel supports multi-input, multi-output transactions, making it perfect for pre-signed mass token dispersal scenarios where a single account must disperse tokens to a large number of unique recipients cost-efficiently.
```

File: src/v1.0.0/1. Introduction/3. Use Cases.md [hunk 19]
```diff
@@ -0,0 +1,62 @@
...
+
+This is possible on Fuel due to a **merklizied dispersal pattern and Fuel's unique pre-signed transaction format**.
+
+Also note, tokens are only dispersed when the user claims them post dispersal event.
```
Comment (adlerjohn) [inline src/v1.0.0/1. Introduction/3. Use Cases.md:37]:
```suggestion
This is possible on Fuel with a Merkleized dispersal pattern and our deterministic transaction identifiers, allowing dispersal transactions to be pre-signed and distributed off-chain. Tokens are only actually dispersed when claimed, greatly reducing overhead costs for unclaimed tokens.
```

File: src/v1.0.0/1. Introduction/3. Use Cases.md [hunk 20]
```diff
@@ -0,0 +1,62 @@
...
+
+For further reading on how we accomplish this for near-zero cost, please see [Token Minting](../4.%20Tools%20and%20Applications/1.%20Token%20Minting.md)
```
Comment (adlerjohn) [inline src/v1.0.0/1. Introduction/3. Use Cases.md:39]:
```suggestion
For further reading on how we accomplish this with low overheads, please see [Token Minting](../4.%20Tools%20and%20Applications/1.%20Token%20Minting.md)
```

File: src/v1.0.0/1. Introduction/3. Use Cases.md [hunk 21]
```diff
@@ -0,0 +1,62 @@
...
+
+Fast Withdrawals
+---
+
+Fuel allows for fast (~10 minute) withdrawals using our unique HTLC output. Liquidity providers on the intended exit chain (e.g. Ethereum or Bitcoin) can allow users on Fuel to withdraw with minimal effort and at low-cost.
```
Comment (adlerjohn) [inline src/v1.0.0/1. Introduction/3. Use Cases.md:44]:
```suggestion
Fuel allows for fast (~10 minute) withdrawals using our unique HTLC output. Liquidity providers on the intended exit chain (e.g. Ethereum or Bitcoin) can allow users on Fuel to withdraw with minimal effort and at low cost.
```

File: src/v1.0.0/1. Introduction/3. Use Cases.md [hunk 22]
```diff
@@ -0,0 +1,62 @@
...
+
+For further reading, please see [Fast Withdrawals](../4.%20Tools%20and%20Applications/0.%20Fast%20Withdrawals.md)
```
Comment (adlerjohn) [inline src/v1.0.0/1. Introduction/3. Use Cases.md:46]:
```suggestion
For further reading, please see [Fast Withdrawals](../4.%20Tools%20and%20Applications/0.%20Fast%20Withdrawals.md).
```

File: src/v1.0.0/1. Introduction/3. Use Cases.md [hunk 23]
```diff
@@ -0,0 +1,62 @@
...
+
+Subscriptions
+---
+
+Fuel's pre-signed transaction format is ideal for creating regular financial subscriptions, where a user can *sign up* to a subscription, pre-signing a set of transactions, and then allowing any service or system to disperse them on a regular bases.
```
Comment (adlerjohn) [inline src/v1.0.0/1. Introduction/3. Use Cases.md:51]:
```suggestion
Fuel's pre-signed transaction format is ideal for creating regular financial subscriptions, where a user can *sign up* to a subscription, pre-signing a set of transactions, and then allowing any service or system to collect payment on a regular basis non-interactively.
```

File: src/v1.0.0/1. Introduction/3. Use Cases.md [hunk 24]
```diff
@@ -0,0 +1,62 @@
...
+
+This is allows for cheap, care free, subscriptions for magazines, blogs, forums and more.
```
Comment (adlerjohn) [inline src/v1.0.0/1. Introduction/3. Use Cases.md:53]:
```suggestion
This is allows for cheap, care-free subscriptions for magazines, blogs, forums, and more.
```

File: src/v1.0.0/1. Introduction/3. Use Cases.md [hunk 25]
```diff
@@ -0,0 +1,62 @@
...
+
+For further reading, please see [Subscriptions](../4.%20Tools%20and%20Applications/2.%20Subscriptions.md)
```
Comment (adlerjohn) [inline src/v1.0.0/1. Introduction/3. Use Cases.md:55]:
```suggestion
For further reading, please see [Subscriptions](../4.%20Tools%20and%20Applications/2.%20Subscriptions.md).
```

File: src/v1.0.0/1. Introduction/3. Use Cases.md [hunk 26]
```diff
@@ -0,0 +1,62 @@
...
+
+Burning
+---
+
+Fuel has a return data output, which can be used to store arbitrary transaction data in the Fuel ORU such as: burn flags, tweets, hashes or specific application data.
+
+The burning of a token by sending it to the null or a voided address can be contextualized around a specific event using the return data output, all within in the same transaction.
```
Comment (adlerjohn) [inline src/v1.0.0/1. Introduction/3. Use Cases.md:62]:
```suggestion
Fuel has a "return data" transaction output, which can be used to store arbitrary data in Fuel such as: burn metadata, social media messages, timestamped hashes, or specific application data.

The burning of a token by sending it to the zero address can be associated with a specific event using the return data output, all within in the same transaction.
```

**Pull request merged**